.rubocop.yml

```diff
@@ -36,3 +36,12 @@ RSpec/ExampleLength:
 
 Rails/UnknownEnv:
   Enabled: false
+
+Style/NumericPredicate:
+  Enabled: false
+
+Style/GuardClause:
+  Enabled: false
+
+Style/IfUnlessModifier:
+  Enabled: false
```

Comment:
Match our settings in PDC Describe which are more sensible: https://github.com/pulibrary/pdc_describe/blob/main/.rubocop.yml